pychromecast/controllers/dashcast.py

```diff
@@ -0,0 +1,62 @@
+"""
+Controller to interface with the DashCast app namespace.
+"""
+from ..config import APP_DASHCAST
+from . import BaseController
+
+
+APP_NAMESPACE = "urn:x-cast:com.madmod.dashcast"
+
+
+class DashCastController(BaseController):
+    """ Controller to interact with DashCast app namespace. """
+
+    # pylint: disable=useless-super-delegation
+    # The pylint rule useless-super-delegation doesn't realize
+    # we are setting default values here.
+    def __init__(self,
+                 appNamespace=APP_NAMESPACE,
```

Comment:
I think it is a good idea to allow others to override the namespace so they can use pychromecast without modifications if they want to deploy their own fork of dashcast. Allowing the app id to be overridden also makes testing a development branch of the receiver easier.